src/search/command_line.cc

```diff
@@ -20,8 +20,7 @@ using namespace std;
 
 NO_RETURN
 static void input_error(const string &msg) {
-    utils::g_log << msg << endl
-                 << usage() << endl;
```

Comment:
If we no longer print usage on a parser error, maybe we can undo the global variable `g_program_name`?